packages/orbit-components/src/Skeleton/Skeleton.stories.tsx

```diff
@@ -18,129 +15,113 @@ enum PRESETS {
   Text = "Text",
 }
 
-export const Default = () => {
-  return <Skeleton height="150px" width="500px" />;
+const meta: Meta<typeof Skeleton> = {
+  title: "Skeleton",
+  component: Skeleton,
 };
 
-Default.story = {
-  parameters: {
-    info: "This is the default configuration of this component. Visit Orbit.Kiwi for more detailed guidelines.",
-  },
-};
+export default meta;
+type Story = StoryObj<typeof Skeleton>;
 
-export const Playground = ({
-  animate,
-  height,
-  maxHeight,
-  rowBorderRadius,
-  rowHeight,
-  rowOffset,
-  rows,
-  spaceAfter,
-  viewBox,
-  width,
-  title,
-  color,
-}) => {
-  return (
-    <Skeleton
-      animate={animate}
-      color={color}
-      height={height}
-      maxHeight={maxHeight}
-      title={title}
-      rowBorderRadius={rowBorderRadius}
-      rowHeight={rowHeight}
-      rowOffset={rowOffset}
-      rows={rows}
-      spaceAfter={spaceAfter}
-      viewBox={viewBox}
-      width={width}
-    />
-  );
-};
+export const Default: Story = {
+  render: () => <Skeleton height="150px" width="500px" />,
```

Comment:
We don't have to define `width` and `height` at all as these params are optional and the component is visible without setting them up (and if yes, these props could be set as args and controls could be enabled from my POV).

Reply:
Good point 👍